l10n/bundle.l10n.it.json

```diff
@@ -0,0 +1,121 @@
+{
+  "The SAS session has closed.": " La sessione SAS è stata chiusa.",
```

Comment:
I have no knowledge of Italian. I saw leading whitespace in several messages in this file. Just want to double check is it intentional?

Reply:
No actually :?, i think this might have been a formatting error when pasting it from the shared word document we used, i'll rectify it